testing/test_sct_deepseg_sc.py

```diff
@@ -25,10 +25,10 @@ def init(param_test):
     Initialize class: param_test
     """
     # initialization
-    default_args = ['-i t2/t2.nii.gz -c t2 -igt t2/t2_seg_manual.nii.gz -qc testing-qc']  # default parameters
-    param_test.file_seg = 't2_seg.nii.gz'
-    param_test.fname_gt = 't2/t2_seg_manual.nii.gz'
-    param_test.dice_threshold = 0.8
+    default_args = ['-i t2/t2.nii.gz -c t2 -qc testing-qc']  # default parameters
+    param_test.file_seg = 't2_seg.nii.gz'  # output segmentation
+    param_test.fname_gt = 't2/t2_seg-manual.nii.gz'
+    param_test.dice_threshold = 0.8  # TODO: change for 1.0
```

Comment:
Do we want to move to "1.0" in this PR?

Reply:
i've actually removed all the integrity test in this file-- i forgot to remove those lines

Reply:
^ here: 8dc6eac8d88e7f092611cdfeac0e97a74a758e8c